docs/cli/edgedb_watch.rst

```diff
@@ -0,0 +1,23 @@
+.. _ref_cli_edgedb_watch:
+
+
+============
+edgedb watch
```

Comment:
I think this command can have a "version added" marker. Or don't we have one?

Reply:
We have one, but this is in part of the documentation that is not versioned. (Technically, it s versioned in that I can drop `versionadded` anywhere, but we've split out the docs sections that are versioned from those that aren't in the UI.)

<img width="268" alt="image" src="https://user-images.githubusercontent.com/2027711/231959925-7f5f273c-0b50-4f30-90a2-dfd97cf4edda.png">

I believe the thinking in not including the CLI in the overall doc versioning is that it has it's own versions that might not map cleanly to EdgeDB versions, so having that content change with the dropdown wouldn't necessarily work. Is that right, @1st1?

Do we need to rethink that or separately version the CLI section of the docs in some way? Maybe we keep using `versionadded` here in CLI but handle it differently? Maybe we just display the version number from `versionadded` but don't actually show or hide content based on the dropdown setting? That way, the version numbers we use in the CLI docs could correlate not to EdgeDB server versions but to the CLI's versions.

Reply:
Yes, I think we can just display version number here but not show/hide. This specific feature depends on EdgeDB 3.0, but we can also have features that don't. And we can introduce features in minor versions of the CLI tools too. For example `migration create --squash` is introduced in CLI 3.0, but by design also works in 2.x (although, I'm not sure which one of 2.x). Even if we port it to 2.x branch the minor version of CLI tool is not in sync with minor version of the server.